### PR TITLE
TA-1_Added gitworkflow package and GitWorkflow class

### PR DESCRIPTION
**Pull request by MarianaLearningAccount**
Added gitworkflow package and GitWorkflow class

### CODE REVIEW

File: src/main/java/com/mariana/gitworkflow/GitWorkflow.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.mariana.gitworkflow;
+
+public class GitWorkflow {
+    public static void main(String[] args ) {
+
+        String step1 = "Perform git stash - to store the uncommitted changes";
+
```
Comment (AndreiSecu) [inline src/main/java/com/mariana/gitworkflow/GitWorkflow.java]: There are missing a lot of details, please add all of them before committing

File: src/main/java/com/mariana/gitworkflow/GitWorkflow.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+
+
```
Comment (AndreiSecu) [inline src/main/java/com/mariana/gitworkflow/GitWorkflow.java:26]: comment 2

File: src/main/java/com/mariana/gitworkflow/GitWorkflow.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package com.mariana.gitworkflow;
+
+public class GitWorkflow {
+    public static void main(String[] args ) {
+
+        String step1 = "Perform git stash - to store the uncommitted changes";
+
+
```
Comment (TestAutomationPanfilElena) [inline src/main/java/com/mariana/gitworkflow/GitWorkflow.java]: There are missing a lot of details

**Review by AndreiSecu — CHANGES_REQUESTED** [review]

**Review by TestAutomationPanfilElena — CHANGES_REQUESTED** [review]

File: src/main/java/com/mariana/gitworkflow/GitWorkflow.java (hunk 2 above)
Comment (MarianaLearningAccount) [reply src/main/java/com/mariana/gitworkflow/GitWorkflow.java:26]: Hello! Please review the new commit. For clarity, its message is "Defined the Git WorkFlow for future development strategy".

File: src/main/java/com/mariana/gitworkflow/GitWorkflow.java (hunk 3 above)
Comment (MarianaLearningAccount) [reply src/main/java/com/mariana/gitworkflow/GitWorkflow.java]: Hello! Please review the new commit. For clarity, its message is "Defined the Git WorkFlow for future development strategy".

**Review by MarianaLearningAccount — COMMENTED** [review]
Review comment / test

**Comment by PascariPetru** [comment]
Super, adauga modificarile din prezentare slide 27 pasii 13-17

**Review by PascariPetru — CHANGES_REQUESTED** [review]
Super, modifica pasii 13-17 inclusiv din prezentare slide 27

**Review by TestAutomationPanfilElena — APPROVED** [review]

**Review by PascariPetru — APPROVED** [review]

**Pull request merged**